src/Pelagos/Bundle/AppBundle/Controller/UI/MaintenanceModeController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Pelagos\Bundle\AppBundle\Controller\UI;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+use Pelagos\Util\MaintenanceMode;
+
+/**
+ * The DIF controller for the Pelagos UI App Bundle.
+ *
+ * @Route("/maintenancemode")
+ */
+class MaintenanceModeController extends UIController
+{
+    /**
+     * The default action for Maintenance Mode.
+     *
+     * @param Request $request The Symfony request object.
+     *
+     * @Route("/")
+     *
+     * @return Response A Response instance.
+     */
+    public function defaultAction(Request $request)
+    {
+        $this->denyAccessUnlessGranted('IS_AUTHENTICATED_FULLY');
```

Comment:
I think we should give access only to DRPMs

Reply:
Yes, have to change that.